neutrona/datadog_checks/neutrona/neutrona.py

```diff
@@ -0,0 +1,142 @@
+import requests
+from requests import RequestException
+
+import json
+
+from datadog_checks.checks import AgentCheck
+from datadog_checks.errors import CheckException
+
+
+class NeutronaCheck(AgentCheck):
+    def check(self, instance):
+
+        neutrona_express_route_api_url = 'https://expressroutetelemetry.neutrona.com'
+        azure_authentication_url = 'https://login.microsoftonline.com'
+        azure_management_url = 'https://management.azure.com/'
+
+        # TESTING
+        try:
+            neutrona_express_route_api_url = instance['azure']['testing']['neutrona_express_route_api_url']
+            azure_authentication_url = instance['azure']['testing']['azure_authentication_url']
+            azure_management_url = instance['azure']['testing']['azure_management_url']
```

Comment:
Can we remove these and have the tests mock things?

Reply:
We prefer to keep it, this way we can mock the cloud provider and our own API using Docker, validating the REST part, it's all REST calls after all.